.gitignore

```diff
@@ -121,6 +121,14 @@ Tools/msi/obj
 Tools/ssl/amd64
 Tools/ssl/win32
 
+# TODO: Once we auto-regen frozem modules for Windows builds
+# we can drop the .h files from the repo and ignore them here.
+# At that point we will rely the frozen manifest file to identify
+# changed generated files.  We'll drop the entry for it then.
+# See: Tools/scripts/freeze_modules.py.
+Python/frozen_modules/MANIFEST
```

Comment:
I had removed it from the repo since I'd added the .h files back in, but now that I think about it it would be good to have it in the repo regardless.  So, yeah, it should not be ignored.

Reply:
This is done.